TicTacToe/README.md

```diff
@@ -6,6 +6,6 @@ The input is a 3*3 matrix that looks like this (as an example):
         [[1, 2, 0],
         [2, 1, 0],
```

Comment:
why did you change this file?

make sure that the pull request you are making contains only the necessary changes.